pkgs/top-level/all-packages.nix

```diff
@@ -15703,6 +15703,9 @@ with pkgs;
   cargo-edit = callPackage ../development/tools/rust/cargo-edit {
     inherit (darwin.apple_sdk.frameworks) Security;
   };
+  cargo-leptos = callPackage ../development/tools/rust/cargo-leptos {
+    inherit (darwin.apple_sdk.frameworks) Security;
```

Comment:
https://github.com/NixOS/nixpkgs/issues/204303

Reply:
Actually that issue is not as clear as I thought it was, so let me explain some more: we'd like to move towards the style of taking `darwin`  as a package input, rather than inheriting things from it in `callPackage`.

Reply:
Actually, I think this might be my misunderstanding.  Ignore this entire thread.  Sorry.

Reply:
No worries, I was just trying to copy cargo-edit, but if you'd prefer a different way I'm fine with that.